pedigroup/model/group.py

```diff
@@ -0,0 +1,25 @@
+from model.exceptions.cannotBeRemovedException import CannotBeRemovedException
+
+class Group:
+
+    def __init__(self, name):
+        self.name = name
+        self.users = set()
+        self.orders = list()
+
+    def addUser(self, user):
+        self.users.add(user)
+
+    def removeUser(self, user):
+        if not self.users.__contains__(user):
+            raise CannotBeRemovedException("No se puede remover un usuario el cual no pertenece al grupo")
+        self.users.remove(user)
+
+    def addOrder(self, order):
+        self.orders.append(order)
+
+    def usersQuantity(self):
+        return self.users.__len__()
+    
+    def ordersQuantity(self):
+        return self.orders.__len__()
```

Comment:
python tiene una función "len" ya incorporada, que es un poco mas comoda para leer/escribir
```suggestion
    def usersQuantity(self):
        return len(self.users)
    
    def ordersQuantity(self):
        return len(self.orders)
```